### PR TITLE
Maint/70 translate class names and ids to english

### PR DESCRIPTION
**Pull request by devtobi**
**Description**
- Translated code (class names, IDs, comments) to english
- Translated constants (e.g. messages) to english
- Translated templates to english

**Reference**
Issues #70

### CODE REVIEW

**Comment by devtobi** [comment]
Info: Still have to do formatting here.

File: refarch-frontend/src/components/common/YesNoDialog.vue [hunk 1]
```diff
@@ -38,14 +38,14 @@
           variant="text"
           @click="no"
         >
-          Nein
+          No
```
Comment (DanielOber) [inline refarch-frontend/src/components/common/YesNoDialog.vue:41]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/components/common/YesNoDialog.vue [hunk 2]
```diff
@@ -38,14 +38,14 @@
...
         </v-btn>
         <v-btn
           id="yesnodialog-btn-yes"
           color="primary"
           @click="yes"
         >
-          Ja
+          Yes
```
Comment (DanielOber) [inline refarch-frontend/src/components/common/YesNoDialog.vue:48]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/components/common/YesNoDialog.vue [hunk 3]
```diff
@@ -56,24 +56,22 @@
...
- * Beispiel:
+ * Example:
  * <yes-no-dialog
  *    v-model="deleteDialog"
- *    buttontext="Löschen"
- *    dialogtitle="Löschen?"
- *    dialogtext="Wollen Sie die Entität wirklich löschen?"
+ *    buttontext="Delete"
+ *    dialogtitle="Delete?"
+ *    dialogtext="Do you really want to delete the entity?"
```
Comment (DanielOber) [inline refarch-frontend/src/components/common/YesNoDialog.vue:74]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/composables/saveLeave.ts [hunk 4]
```diff
@@ -1,23 +1,22 @@
...
 
 import { ref } from "vue";
 import { onBeforeRouteLeave } from "vue-router";
 
 export function useSaveLeave(isDirty: () => boolean) {
-  const saveLeaveDialogTitle = ref("Ungespeicherte Änderungen");
+  const saveLeaveDialogTitle = ref("Unsaved changes");
   const saveLeaveDialogText = ref(
-    "Es sind ungespeicherte Änderungen vorhanden. Wollen Sie die Seite verlassen?"
+    "There are unsaved changes. Do you want to leave the page?"
```
Comment (DanielOber) [inline refarch-frontend/src/composables/saveLeave.ts:19]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/views/GetStartedView.vue [hunk 5]
```diff
@@ -2,14 +2,14 @@
   <v-container>
     <v-row class="text-center">
       <v-col>
-        <h1 class="text-h3 font-weight-bold mb-10">Dokumentation</h1>
-        <h3>Weiterführende Links sind hier zu finden:</h3>
+        <h1 class="text-h3 font-weight-bold mb-10">Documentation</h1>
+        <h3>Further links can be found here:</h3>
```
Comment (DanielOber) [inline refarch-frontend/src/views/GetStartedView.vue:6]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/views/GetStartedView.vue [hunk 6]
```diff
@@ -45,16 +45,16 @@
...
           >
         </div>
       </v-col>
     </v-row>
     <yes-no-dialog
       v-model="saveLeaveDialog"
-      dialogtitle="Wirklich verlassen?"
-      dialogtext="Wollen Sie wirklich nicht die Dokumentation anschauen?"
+      dialogtitle="Do you want to leave?"
+      dialogtext="Are you sure you don't want to read the documentation?"
```
Comment (DanielOber) [inline refarch-frontend/src/views/GetStartedView.vue:57]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/views/HomeView.vue [hunk 7]
```diff
@@ -11,10 +11,10 @@
 
       <v-col class="mb-4">
         <h1 class="text-h3 font-weight-bold mb-3">
-          Willkommen beim RefArch-Kickstarter
+          Welcome to the default refarch-frontend by it@M
         </h1>
         <p>
-          Das API-Gateway ist:
+          The API-Gateway is:
```
Comment (DanielOber) [inline refarch-frontend/src/views/HomeView.vue:17]: Dont change displayed text, that is visible to the user to english.

File: refarch-frontend/src/api/error.ts [hunk 8]
```diff
@@ -8,18 +8,18 @@ export class ApiError extends Error {
   level: string;
   constructor({
     level = Levels.ERROR,
-    message = "Ein unbekannter Fehler ist aufgetreten, bitte den Administrator informieren.",
+    message = "An unknown error has occurred, please inform the administrator.",
```
Comment (DanielOber) [inline refarch-frontend/src/api/error.ts:11]: Dont change displayed text, that is visible to the user to english.

**Review by DanielOber — APPROVED** [review]
LGTM

**Pull request merged**